react/Badge/Badge.js

```diff
@@ -25,7 +25,7 @@ export default class Badge extends PureComponent<Props> {
   props: Props;
 
   render() {
-    const { children, tone, strong } = this.props;
+    const { children, tone = '', strong } = this.props;
```

Comment:
Should this be defaulting to `neutral`?

Reply:
Yeah could do. My aim was for it to not default to undefined. Happy to change it to neutral instead

Reply:
Well really, we could say that `tone` is not optional and default it, that might be a more accurate contract. Does that make sense?

Reply:
We've already had it with Highlight, seems sensible to default to neutral

Reply:
Yeah that's probably more accurate here. Away from my laptop but I'll make tone required later this arvo.

Reply:
https://github.com/seek-oss/seek-style-guide/blob/master/react/Highlight/Highlight.js#L15